### DISCUSSION TITLE
Python bindings

### DISCUSSION THREAD

**Discussion by oleg-kachan** [vlang/v#11611] [Ideas]
It would be nice to see V/Python interoperability, for example, similar to pybind.

**Comment by oleg-kachan** [comment]
Anything changed in this direction since 2019?

**Reply by JalonSolov** [reply]
V supports full C interop.  Python has C interop.  Just match the 2 together.

There are no plans to add Python specific "bindings" to V.

**Reply by oleg-kachan** [reply]
Thanks!

Yet, could you or anyone provide a minimal code example how can I pass data from Python to V and get the result back? It would be highly appreciated!

**Reply by JalonSolov** [reply]
This might help... https://github.com/SaptakBhoumik/call-v-from-python
